BenchTesting.java

```diff
@@ -62,5 +65,87 @@ public static void main(String[] args) {
         System.out.println("Expected: temp1. Actual: " + island1.getPassDetails(123));
         System.out.println("Expected: true. Actual: " + island1.isPassOnIsland(pass1));
         System.out.println("Expected: false. Actual: " + island1.isPassOnIsland(pass2));
+//Testing for Ferry class-> To test everything for this class, you need to comment the Island and Pass variables above ^
```

Comment:
Why does it need to be commented?

Reply:
To make it easier for you to read

Reply:
Just not sure why there's a massive block of code that is commented. What was it's purpose and why is it commented?

Reply:
no worries i reformatted it again, i hope its cleaner now